doc/tutorials/notifications.md

```diff
@@ -0,0 +1,327 @@
+Preface
+ ===
+ 
+ **The features described in this document are not
+implemented yet or experimental. This document contains a preview of
+how developers would use the notification feature**
+ 
+ This document
+explains how notifications are implemented in Elektra and how they
+can be used by application developers.
+ 
+ Notifications - Overview &
+Concept
+ ===
+ 
+ Elektra's notification feature consists of several
+components.
+ While notification features are implemented by plugins,
+applications use APIs provided by wrappers in order to use different
+plugins.
+ 
+ A wrapper for notifications provides the API for receiving
+and handling notifications.
+ A wrapper for I/O operations allows
+asynchronous notification processing by compatible plugins.
+ The
+I/O-wrapper consists of an *interface* used by transport plugins
+and a *binding* which implements the actual I/O management functions
+for a specific I/O management library.
+ 
+ Transport plugins exchange
+notifications via different protocols like D-Bus, Redis and ZeroMQ.
+ For
+each type of transport there are two types of plugins: One for sending
+and one for receiving notifications.
+ Developers do not interact with
+those plugins directly.
+ The underlying transports are transparent
+to them.
```

Comment:
to?

Reply:
I think the preposition is right. I found [a discussion on dict.leo.org](https://dict.leo.org/forum/viewWrongentry.php?idThread=1233522&idForum=3&lang=de&lp=ende) and [this page](https://lingohelp.me/preposition-after-adjective/transparent-about-to-in-with-for/).